### PR TITLE
Get total usage per project from db

### PR DESCRIPTION
**Pull request by valyo**
For each unit, get the total usage for every project in bhours from the database. The total cost in SEK per project is also fetched, in case we will need it in the charging model implementation.

Please include the following in this section

- [x] Summary of the changes and the related issue
- [x] Relevant motivation and context
- [ ] Any dependencies that are required for this change

Fixes # DDS-1286

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

Please delete options that are not relevant.

- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Rebase/merge the branch which this PR is made to
- [ ] Changes to the database schema: A new migration is included in the PR

## Formatting and documentation

- [x] I have added a row in the [changelog](../CHANGELOG.md)
- [x] The code follows the style guidelines of this project: Black / Prettier formatting
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

## Tests

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1182"></a>

### CODE REVIEW

**Comment by inaod568** [comment]
@valyo Could you add a short description of what that task it is and move the "FIxes DDS-1286" to where it says "Fixes # (issue)"? 🙏🏻

**Comment by valyo** [comment]
> @valyo Could you add a short description of what that task it is and move the "FIxes DDS-1286" to where it says "Fixes # (issue)"? 🙏🏻

done
I checked the changelog box since we have a entry about this cron job, which we will edit in the end

File: dds_web/scheduled_tasks.py [hunk 1]
```diff
@@ -240,3 +240,28 @@ def monthly_usage():
...
+                for project in page_query(
+                    db.session.query(models.Project)
+                    .filter(
+                        sqlalchemy.and_(
+                            models.Project.is_active == 1, models.Project.unit_id == unit.id
+                        )
+                    )
+                    .with_for_update()
+                ):
+                    proj_bhours, proj_cost = UserProjects.project_usage(project)
```
Comment (inaod568) [inline dds_web/scheduled_tasks.py:257]: Just adding this 
```suggestion
                    proj_bhours, proj_cost = UserProjects.project_usage(project=project)
```

**Review by inaod568 — APPROVED** [review]
Looks good, and works as it should.

**Pull request merged**